app/controllers/categories_controller.rb

```diff
@@ -650,6 +662,9 @@ def fetch_topic_list(topics_filter: nil)
           SiteSetting.top_page_default_timeframe.to_sym,
         )
       @topic_list.more_topics_url = url_for(top_path)
+    when "categories_and_hot_topics"
```

Comment:
Is this necessary for adding the `categories_and_hot` endpoint? This looks like it's only used by the `index` action, is that correct?

Reply:
`categories_and_topics` from line 496 also uses this and `categories_and_topics` is called by `categories_and_hot` from line 85.